core/store.go

```diff
@@ -197,3 +197,28 @@ func countClients(clients *sync.Map) int {
 	})
 	return count
 }
+
+func GetNoTouch(k string) *Obj {
```

Comment:
The function is very similar to `Get` except that we don't update the `v.LastAccessedAt` in this function. I would prefer to add `touch/no touch` feature to the existing `Get` function by adding another parameter to the `Get` function or via variadic arguments.
@soumya-codes what do you think?

Reply:
Hi @gauravsarma1992, this was exactly my approach. Adding a flag parameter would require touching Get(k) at many places. I have updated the function with no Del(k) operation and will work on further comments as progress.

Reply:
Yes, to avoid touching the `GET` function call in many places, we can use variadic arguments. However, variadic arguments are not easy to maintain in the long term.
Ideally, all the store level functions should define their own request objects (k string, shouldUpdateLastAccessAt) so that we don't have to modify function parameters. But that is out of scope of this task.